rpcq/test/test_auth.py

```diff
@@ -161,8 +162,11 @@ def client_auth(request, m_endpoints):
     request.addfinalizer(client.close)
     return client
 
-
-OOPS_VALUE_ERROR_STR = "ValueError('Oops.',)\nTraceback (most recent call last):\n  "
+if sys.version_info < (3, 7):
+    OOPS_VALUE_ERROR_STR = "ValueError('Oops.',)\nTraceback (most recent call last):\n  "
+else:
+    # The default repr for BaseException was changed in 3.7 to elide the trailing comma.
+    OOPS_VALUE_ERROR_STR = "ValueError('Oops.')\nTraceback (most recent call last):\n  "
```

Comment:
Alternatively we could relax the test so this version check isn't required.

We do a similar version check in `_base.py` to see whether we need to vendor `dataclasses`, e.g.